scripts/globals/status.lua

```diff
@@ -1339,8 +1339,8 @@ xi.mod =
     MAGIC_DAMAGE                    = 311, --  Magic damage added directly to the spell's base damage
 
     -- Gear set modifiers
-    DA_DOUBLE_DAMAGE                = 408,  -- Double attack's double damage chance %.
-    TA_TRIPLE_DAMAGE                = 409,  -- Triple attack's triple damage chance %.
+    DA_DOUBLE_DMG_RATE              = 408,  -- Double attack's double damage chance %.
```

Comment:
missed this in modifier.h 

also I see other edits in modifier.h that i am confused by: every time someone adds a mod, it should be both there and in status.lua - were these other mods _MISSING?_

Reply:
To my knowledge the mods were not outright missing from either status.lua or modifier.h and only had to have their names updated. The CHAKRA_MULT and CHAKRA_REMOVAL mods in modifier.h I only removed a space before the comment to align them with the other mods nearby.

Reply:
I was so confused by seeing "modifier.h" in the documents NAME

Reply:
Those were missing from documentation/modifier.h, but they existed in src/map/modifier.h and scripts/globals/status.lua.